docs/mocksupport.rst

```diff
@@ -1,7 +1,7 @@
 .. _`Mock support`:
 
-Mock support
-============
+Mock WBEM server support
```

Comment:
"Mock WBEM server support" sounds a bit strange to me, but then I'm not a native speaker. How about "Mock WBEM server" or "WBEM server mock support"?

Also, please change the link target two lines up accordingly. If you want to provide a compatible link target, you can have two link targets.

Reply:
Much better.  I went off the cliff when I first used the word support.